### PR TITLE
Add Kserve UI tests

### PR DESCRIPTION
**Pull request by lugi0**
Testing in rhods-ci-pr-test/2178

Adds a set of basic tests for Kserve deployed models using the RHOAI UI

### CODE REVIEW

**Comment by tarukumar** [comment]
Generally, it appears good. However, we might consider updating the keyword and refining the inference related to the workbench name, even when we select all projects in the dropdown. This could be enhance we can do in the future

**Comment by lugi0** [comment]
@jstourac no I will update with the correct Polarion IDs ASAP before merging

**Review by jstourac — COMMENTED** [review]
I have very little knowledge about the topic, but in general this LGTM (even though, there are a handful of robot warnings that could be easily fixed) :upside_down_face: :slightly_smiling_face:

**Comment by lugi0** [comment]
Validated once in rhods-ci-pr-test/2187, rerunning in rhods-ci-pr-test/2188 after last commit to ensure nothing broke

**Comment by lugi0** [comment]
Obviously everything broke :) Retesting for hopefully the last time: rhods-ci-pr-test/2189

**Comment by lugi0** [comment]
verified in rhods-ci-pr-test/2191

File: ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/422__model_serving_llm_UI.robot [hunk 1]
```diff
@@ -0,0 +1,572 @@
...
+    Set Suite Variable    ${EXP_RESPONSES}    ${exp_responses}
+
+Set Up Project
+    [Documentation]    Creates the DS Project (if not exists), creates the data connection for the models,
+    ...                creates caikit runtime. This can be used as test setup
+    [Arguments]    ${namespace}    ${single_prj}=${TRUE}    ${enable_metrics}=${FALSE}    ${dc_name}=kserve-connection
+    Open Model Serving Home Page
+    IF    ${single_prj}
+        Try Opening Create Server
+        Wait for RHODS Dashboard to Load    wait_for_cards=${FALSE}    expected_page=Data Science Projects
```
Comment (bdattoma) [inline ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/422__model_serving_llm_UI.robot:295]: why do we need to handle this case?

Comment (lugi0) [reply ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/422__model_serving_llm_UI.robot:295]: `Try Opening Create Server` deletes all existing projects to test with only a single DS Project present - This way we can force multiple projects at the same time by specifying `singel_prj=${FALSE}`

**Pull request merged**